### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by ecl996**
#### What this PR does / why we need it:

Minor spelling mistake in the file

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -69,7 +69,7 @@ sts:
 
 properties:
   # use externalSecure for when inbound SSL is provided by nginx-ingress or other external mechanism
-  sensitiveKey: changeMechangeMe # Must to have minnimal 12 length key
+  sensitiveKey: changeMechangeMe # Must to have minimal 12 length key
   alogorithm: NIFI_PBKDF2_AES_GCM_256
```
Comment (banzo) [inline values.yaml:73]: there is one in `alogorithm` too I am guessing

Comment (ecl996) [reply values.yaml]: Ah yes! Added to the PR

**Review by zakaria2905 — CHANGES_REQUESTED** [review]
Could you please update the chart version to 1.0.2 in the Chart.yaml file

**Comment by ecl996** [comment]
> Could you please update the chart version to 1.0.2 in the Chart.yaml file

Yup, all set!

**Review by zakaria2905 — APPROVED** [review]
Your changes have been accepted and will be merged, thanks.

**Pull request merged**